src/app/pages/homepage/parts/quickstart/homepage-quickstart.component.html

```diff
@@ -6,18 +6,18 @@ <h2 class="mb-6 text-3xl font-extrabold tracking-tight text-gray-900 dark:text-w
           <transloco key="quickstart" />
         </h2>
 
-        <p class="mb-4">
-          <transloco key="quickstart.register_account" [params]="{registerLink: {route: '/register'}}" />
+        <p class="mb-4" [innerHTML]="'quickstart.register_account' | transloco" >
+          
```

Comment:
This would be better reverted and the `registerLink` changed from `{route: '/register'}` to just `/register`, that way it will use href instead of routerLink.

Reply:
If you can demonstrate it works the same, that sounds fine to me